app/views/support/recruitment_cycles/index.html.erb

```diff
@@ -2,36 +2,34 @@
 
 <div class="govuk-width-container">
   <div class="govuk-grid-row">
```

Comment:
We should remove this div. As far as I'm aware, the `govuk-grid` classes work similarly to Bootstrap, which the `row` class adds negative margins.

Reply:
Sounds good!